### ISSUE TITLE
Saving to the calendar

### ISSUE THREAD

**Issue by IraSoro** [IraSoro/peri#142]
Try saving changes to the calendar without the ```Save``` button.

**Comment by IraSoro** [comment]
It won’t be possible to do this yet; the Ionic has limitations.

**Comment by imblowfish** [comment]
I'd like to add specific information about on what limitations we encountered in the current version of IonDatetime
IonDatetime has `onChange` callback which we used. In case if new dates is differs from previous values we called`setShowSaveButton` and new `button` was mounted to the IonDatetime as a child
It looks something like this:
```html
<button>...</button>
<input value="...">...</input>
``` 
Also, in the `onChange` callback we receiving the current values of `input` element. As example they may looks like this

```js
["2023-10-27","2023-10-28"]
```
When we click on any date in IonDatetime again, `onChange` will be called again, but we don't get any new dates as argument. For example, if IonDatetime had 2 values as parameter on the previous render, then the same parameters will be on the next render after `setShowSaveButton` has been changed. We'll get new values only after the next `button` mount. In this case new values will be appeared in the DOM. This behavior of IonDatetime is only present if we change the state in the `onChange` callback
